hw/ip/otp_ctrl/data/dif_otp_ctrl_unittest.cc.tpl

```diff
@@ -39,6 +39,31 @@ class OtpTest : public testing::Test, public MmioTest {
   dif_otp_ctrl_t otp_ = {.base_addr = dev().region()};
 };
 
+class DaiRegwenTest : public OtpTest {};
+
+TEST_F(DaiRegwenTest, LockDai) {
```

Comment:
Add checks for kBadArgs when passing nullptr instead of &otp_ for both functions, and also when &flag is nullptr.

Reply:
added some more checks!